### ISSUE TITLE
RT-Dino

### ISSUE THREAD

**Issue by sebbyjp** [MbodiAI/mbodied-agents#33]
Resources
[Highly-Recommended Guide to Follow](https://karpathy.github.io/2019/04/25/recipe/?fbclid=IwAR0HcxdKV2lPGYGKTZI7x1wkNAnxfp7hvNHAHPJl2CtxAK26w44d76n37ig)

https://github.com/lucidrains/vit-pytorch?tab=readme-ov-file#3d-vit
[DinoV2 Source Code](https://github.com/facebookresearch/dinov2)
https://github.com/lucidrains/classifier-free-guidance-pytorch

[MinMax Scaler](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.MinMaxScaler.html#minmaxscaler)
[kbins](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.KBinsDiscretizer.html)

DinoV2 backbone, multihead transformer head, classifier free film layers, xarm data
- [ ] Use ViT-g/14 with registers 
- [ ] Asymmetric loss: https://timm.fast.ai/asymmetric_loss
- [ ] CutUpMix: https://timm.fast.ai/random_resized_crop
- [ ] https://timm.fast.ai/random_resized_crop
- [ ] [PCA] (https://scikit-learn.org/stable/modules/generated/sklearn.decomposition.PCA.html#sklearn.decomposition.PCA)

--

<br class="Apple-interchange-newline">
